prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -1,120 +1,167 @@
+              hostPath:
+                path: /var/lib/dbus
             - name: lib-modules
-              mountPath: /lib/modules
-          resources:
-            requests:
-              cpu: 10m
-        - name: weave-npc
-          image: weaveworks/weave-npc:latest
-          imagePullPolicy: Always
-          resources:
-            requests:
-              cpu: 10m
+              hostPath:
+                path: /lib/modules
+          hostPID: true
+          hostNetwork: true
+          serviceAccountName: weave-net
+          restartPolicy: Always
           securityContext:
-            privileged: true
-      restartPolicy: Always
-      tolerations:
-      - key: node-role.kubernetes.io/master
-        effect: NoSchedule
-      serviceAccountName: weave-net
-      securityContext:
-        seLinuxOptions:
-          type: spc_t
-      volumes:
-        - name: weavedb
-          emptyDir: {}
-        - name: cni-bin
-          hostPath:
-            path: /opt
-        - name: cni-bin2
-          hostPath:
-            path: /home
-        - name: cni-conf
-          hostPath:
-            path: /etc
-        - name: dbus
-          hostPath:
-            path: /var/lib/dbus
-        - name: lib-modules
-          hostPath:
-            path: /lib/modules
+            seLinuxOptions:
+              type: spc_t
+    apiVersion: extensions/v1beta1
+    kind: DaemonSet
+  - metadata:
+      labels:
+        name: weave-net
+      name: weave-net
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "server-version": "master-c1f3803",
+            "original-request": {
+              "url": "/k8s/v1.6/net?v=latest",
+              "date": "Thu Apr 20 2017 09:43:20 GMT+0000 (UTC)"
+            },
+            "email-address": "support@weave.works"
+          }
+    rules:
+      - apiGroups:
+          - ''
+        resources:
+          - pods
+          - namespaces
+          - nodes
+        verbs:
+          - get
+          - list
+          - watch
+      - apiGroups:
+          - extensions
+        resources:
+          - networkpolicies
+        verbs:
+          - get
+          - list
+          - watch
+    apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: ClusterRole
+  - metadata:
+      labels:
+        name: weave-net
+      name: weave-net
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "server-version": "master-c1f3803",
+            "original-request": {
+              "url": "/k8s/v1.6/net?v=latest",
+              "date": "Thu Apr 20 2017 09:43:20 GMT+0000 (UTC)"
+            },
+            "email-address": "support@weave.works"
+          }
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: weave-net
+    subjects:
+      - kind: ServiceAccount
+        name: weave-net
+        namespace: kube-system
+    apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: ClusterRoleBinding
```

Comment:
We should change this in the Launch Generator directly then. Cc: @errordeveloper.
Happy to do it, but after I'm done with the proxy merge.